src/sidebar/util/view-filter.js

```diff
@@ -0,0 +1,195 @@
+/**
```

Comment:
I _did_ `git mv` this, but git seems to think this is a wholly new file. I did restructure the former service a little to pull a bunch of functions and classes outside of the wrapping function that has here been renamed `viewFilter`.

Reply:
I was under the impression that git has no concept of "moves" in how it represents the contents of a commit but instead this is inferred when doing `git diff` or `git log` from a removed and an added file in a commit having a certain percentage overlap in their contents. If this is correct, then changing a file beyond this threshold would cause it not to be detected as a rename.